3-Authorization-II/protect-web-api/AppCreationScripts/Configure.ps1

```diff
@@ -13,7 +13,7 @@ param(
 
  In case you don't have Microsoft.Graph.Applications already installed, the script will automatically install it for the current user
  
- There are four ways to run this script. For more information, read the AppCreationScripts.md file in the same folder as this script.
+ There are two ways to run this script. For more information, read the AppCreationScripts.md file in the same folder as this script.
```

Comment:
the `sample.json` that was used to generate these scripts should also be committed